CHANGELOG.md

```
@@ -0,0 +1,34 @@
+
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Fixed
+
+### Removed
+
+[2025.10.03]: https://github.com/Duncan1106/item-list-card/releases/tag/2025.10.03
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)